app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -39,6 +39,8 @@ def submit
     redirect_to confirmation_url(reference_number: submission_reference, contact_gp: @contact_gp)
   end
 
+  EXCLUDED_FIELDS = %i[session_id _csrf_token current_path previous_path check_answers_seen postcode].freeze
```

Comment:
Should `postcode` be removed from the session in postcode lookup controller instead? Can `support_address.postcode` be used for all of the lookups instead?

Reply:
You are right, I will close this and change the controller in another PR. We should probably also go from excluding fields from the session to including them, perhaps using keys from the schema? :)